### PR TITLE
LPD-20585 Add Friendly URL to documents Headless API

### PR DESCRIPTION
**Pull request by Mikellorza**
story: https://liferay.atlassian.net/browse/LPD-19314
subtask: https://liferay.atlassian.net/browse/LPD-20585

Description
-------------
Add friendly url to document headless API.

### CODE REVIEW

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/DocumentDTOConverter.java [hunk 1]
```diff
@@ -162,6 +164,20 @@ public Document toDTO(DTOConverterContext dtoConverterContext)
 				setExternalReferenceCode(fileEntry::getExternalReferenceCode);
 				setFileExtension(fileEntry::getExtension);
 				setFileName(fileEntry::getFileName);
+				setFriendlyUrlPath(
+					() -> {
+						FriendlyURLEntry friendlyURLEntry =
```
Comment (jkappler) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/DocumentDTOConverter.java:174]: @Mikellorza, in the story the display page is mentioned. So just wondering which URL we should expose

Comment (Mikellorza) [reply modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/DocumentDTOConverter.java:174]: But, to double check we can ask Ruth to confirm it.

**Review by jkappler — APPROVED** [review]

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward

**Comment by Mikellorza** [comment]
ci:forward

**Comment by Mikellorza** [comment]
ci:report:2552310632

**Comment by Mikellorza** [comment]
ci:report:2552489467

**Comment by Mikellorza** [comment]
ci:forward

**Comment by Mikellorza** [comment]
ci:report:2552774887

**Comment by Mikellorza** [comment]
I have reviewed some test running reports and I am not seeing anything failing tests related to this PR. So let is move to brian.

**Comment by Mikellorza** [comment]
ci:forward:force

**Pull request closed without merging**